modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ToolbarDrawerToggleTest.ts

```diff
@@ -107,4 +107,50 @@ describe('browser.tinymce.themes.silver.editor.toolbar.ToolbarDrawerToggleTest',
       });
     });
   });
+
+  context(`Should preserve focus if skipFocus: true option was passed`, () => {
+    Arr.each<ToolbarMode>([ 'floating', 'sliding' ], (toolbarMode) => {
+      it(`TINY-9337: Preserves focus in ${toolbarMode} if skipFocus is true`, async () => {
+        const editor = await McEditor.pFromSettings<Editor>({
+          menubar: false,
+          statusbar: false,
+          width: 200,
+          toolbar_mode: toolbarMode,
+          base_url: '/project/tinymce/js/tinymce'
+        });
+        await UiUtils.pWaitForEditorToRender();
+        editor.focus();
+        const initialFocusedElement = document.activeElement;
+        editor.execCommand('ToggleToolbarDrawer', false, { skipFocus: true });
+        await TinyUiActions.pWaitForUi(editor, '.tox-toolbar__overflow');
+        assert.equal(initialFocusedElement, document.activeElement, 'Focus should be preserved');
```

Comment:
Generally, you would use `FocusTools` instead of `document.activeElement` directly. This is especially true as it simplifies the path forward to more compete shadow DOM Support. There are also convenience functions (e.g. `pTryOnSelector`) that can make this a bit more streamlined.

- [ ] switch to using FocusTools where possible

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8272/commits/3eb16a0e34d89b19e5ace1e84413596938211b85).